### ISSUE TITLE
Bug: NPM install fails on windows due to post-install script from fontawesome-common-types

### ISSUE THREAD

**Issue by deepu105** [FortAwesome/Font-Awesome#19243]
### Bug description

When installing on windows, NPM install fails with the below error

```
9170 info run @fortawesome/fontawesome-svg-core@6.1.2 postinstall { code: 127, signal: null }
9171 timing reify:rollback:createSparse Completed in 15769ms
9172 timing reify:rollback:retireShallow Completed in 0ms
9173 timing command:i Completed in 108992ms
9174 verbose stack Error: command failed
9174 verbose stack     at ChildProcess.<anonymous> (C:\Program Files\nodejs\node_modules\npm\node_modules\@npmcli\promise-spawn\lib\index.js:63:27)
9174 verbose stack     at ChildProcess.emit (node:events:527:28)
9174 verbose stack     at maybeClose (node:internal/child_process:1092:16)
9174 verbose stack     at Process.ChildProcess._handle.onexit (node:internal/child_process:302:5)
9175 verbose pkgid @fortawesome/fontawesome-common-types@6.1.2
9176 verbose cwd C:\Users\d4udt\workspace\jhtest
9177 verbose Windows_NT 10.0.19044
9178 verbose node v16.16.0
9179 verbose npm  v8.11.0
9180 error code 127
9181 error path C:\Users\d4udt\workspace\jhtest\node_modules\@fortawesome\fontawesome-common-types
9182 error command failed
9183 error command bash -c node attribution.js
9184 error /bin/bash: node: command not found
9185 verbose exit 127
```

### Reproducible test case

none since its an NPM install issue

### Font Awesome version

6.1.2

### Serving

Other (as specified in the bug description)

### Implementation

SVG+JS

### Browser and Operating System

Windows

### Web bug report checklist

- [X] I have included a test case because my odds go _way_ up that the team can fix this when I do
- [X] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate

**Comment by tagliala** [comment]
Hi,

I think this issue is the same as #18803, and the fix is planned for 6.2.0

Could you please confirm?

**Comment by deepu105** [comment]
The issue we faced seems to be due ro presence of an npmrc file that forced using bash. So I'll close this
